test/promise.ts

```diff
@@ -15,7 +15,9 @@ describe('NodeID3.Promise', function () {
             ).to.eventually.be.instanceOf(Buffer)
         })
     })
-    const writeTestCases = [
+    type writeTestCaseFunctions =
+        (tags: NodeID3.WriteTags, buffer: Buffer | string) => void
+    const writeTestCases: Array<[string, writeTestCaseFunctions]> = [
```

Comment:
All this change is not necessary, the proper way to fix this is convert the arrays into tuples so the types will be correct, the only thing to change is to add `as const` at the end of the array definition, this will make the definition a tuple instead of an array.